packages/connect/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Changed
+
+- Stop using dynamic imports in order to import the "well-known" specs, due to the fact that not all bundlers are able to handle dynamic imports gracefully on package's code.
+
```

Comment:
This is no needed information for the users. I think we can remove it

Reply:
Sorry, I'm not following. What information is not needed? What is that you think that it should be removed? Could you please propose an alternative text?

Reply:
No worries - Lets just keep that as is 👍🏽